docs/source/basic_tutorial.rst

```diff
@@ -195,6 +195,7 @@ Now that we have our our agent, we'll set up the display loop.
         if world.epoch_done():
             print('EPOCH DONE')
             break
+    world.shutdown()
```

Comment:
I'm intentionally leaving this out of the tutorial, as it's only needed in very specific cases, not in the general case (e.g. with a ZMQ connection that should send a shutdown signal for a remote agent).

In a similar vein, this example also doesn't explain more advanced features like the 'share' function which enables batching or hogwild training.